src/api/uploadFile/Uploadfile.ts

```diff
@@ -0,0 +1,9 @@
+import { accessClient } from 'api';
+
+export const uploadFile = (formData: FormData) => accessClient.post('/OWNERS/upload/file', formData, {
+  headers: { 'Content-Type': 'multipart/form-data' },
+});
```

Comment:
반환값이 정해져야 해당 데이터를 명확히 사용할 수 있을거예요. (지금은 data?.data?. ... 이렇게 쓰고있네요)
두번째 제네릭이엇나는 정확하지않음 ㅎ
```suggestion
export const uploadFile = (formData: FormData) => accessClient.post<{
  "expiration_date": string;
  "file_url": string;
  "pre_signed_url": string;
}>('/OWNERS/upload/file', formData, {
  headers: { 'Content-Type': 'multipart/form-data' },
});
```

Reply:
다음 pr에서 수정해보도록 하겠습니다.!